src/ddscxx/src/org/eclipse/cyclonedds/pub/qos/DataWriterQosDelegate.cpp

```diff
@@ -187,6 +195,7 @@ DataWriterQosDelegate::ddsc_qos(const dds_qos_t* qos)
     strength_    .delegate().set_iso_policy(qos);
 #endif  // OMG_DDS_OWNERSHIP_SUPPORT
     lifecycle_   .delegate().set_iso_policy(qos);
+    datarepresentation_.delegate().set_iso_policy(qos);
```

Comment:
Please also encapsulate this line with the conditional OMG_DDS_EXTENSIBLE_AND_DYNAMIC_TOPIC_TYPE_SUPPORT macro.

Reply:
Will do.